dbms/src/Storages/Page/V3/WAL/serialize.cpp

```diff
@@ -40,6 +40,7 @@ inline void serializeEntryTo(const PageEntryV3 & entry, WriteBuffer & buf)
     writeIntBinary(entry.file_id, buf);
     writeIntBinary(entry.offset, buf);
     writeIntBinary(entry.size, buf);
+    writeIntBinary(entry.align_size, buf);
```

Comment:
@JaySon-Huang is that make sense to you?

Reply:
This is OK since we don't have any release version with PageStorage V3 running.